src/renderer/components/watch-video-info/watch-video-info.js

```diff
@@ -227,9 +227,9 @@ export default Vue.extend({
 
     dateString() {
       const date = new Date(this.published)
-      const dateSplit = date.toDateString().split(' ')
-      const localeDateString = `Video.Published.${dateSplit[1]}`
-      return `${this.$t(localeDateString)} ${dateSplit[2]}, ${dateSplit[3]}`
+      const localeCode = this.$t('Locale Code')
```

Comment:
Instead of updating every yaml file, you could have just added a computed property like this https://github.com/FreeTubeApp/FreeTube/blob/f4e5537397c0b43e121c4e80f3d3c3cd386a5c6a/src/renderer/components/ft-video-player/ft-video-player.js#L143 to watch-video-info and replaced the underscores with dashes to get the locale code

Reply:
Changed. Thanks